runtimes/node-17.0/README.md

```diff
@@ -0,0 +1,77 @@
+# Node Runtime 17.0
+
+This is the Open Runtime that builds and runs NodeJS code based on a `node:17-alpine` base image. 
+
+The runtime itself uses [Micro](https://github.com/vercel/micro) as the Web Server to process the execution requests.
```

Comment:
The readme looks good but I think you should include a few more points in this section about the folder structure. 
You can briefly mention the purpose of the files and folders here like

```
example/
launch.sh
server.js
```

Or any other files that are not obvious.